src/components/UserLogin.jsx

```diff
@@ -0,0 +1,33 @@
+import { useContext } from "react";
+import CurrentUserContext from "../contexts/CurrentUserContext";
+import { Link } from "react-router-dom";
+
+export default function UserLogin() {
+  const { currentUser, SetCurrentUser } = useContext(CurrentUserContext);
+
+  function handleLogin(event) {
+    event.preventDefault();
+  }
+
+  function handleUsernameChange(event) {
+    event.preventDefault();
+    SetCurrentUser(event.target.value);
+  }
+
+  return (
+    <>
+      <form onSubmit={handleLogin}>
+        <label htmlFor="input-username">Username:</label>
+        <input
+          id="input-username"
+          type="text"
+          value={currentUser}
+          onChange={handleUsernameChange}
+        ></input>
+        <Link to={`/home/${currentUser}`}>
+          <button>Submit</button>
+        </Link>
+      </form>
+    </>
+  );
```

Comment:
This also isn't on the current task - but looks good. You don't **have** to authenticate a user, if you provide the client with users that exist in the database as options rather than having an input box you will avoid this possibility. But you can authenticate if you like